### PR TITLE
Do handle the "What's cooking" mails correctly

### PR DESCRIPTION
**Pull request by dscho**
In reacting to the review of https://github.com/gitgitgadget/gitgitgadget/pull/458, I managed to break the central part of it. D'oh.

### CODE REVIEW

**Review by derrickstolee — APPROVED** [review]

File: lib/mail-archive-helper.ts [hunk 1]
```diff
@@ -117,8 +117,9 @@ export class MailArchiveGitHelper {
             if (!sousChef.messageID) {
                 throw new Error(`Could not parse Message-ID of ${mbox}`);
             }
+            console.log(`Handling "${sousChef.subject}"`);
```
Comment (webstech) [inline lib/mail-archive-helper.ts:120]: Just noting that in the unlikely event `subject` is not found the log will say:

```
  console.log
    Handling "undefined"
```

File: lib/mail-archive-helper.ts [hunk 2]
```diff
@@ -117,8 +117,9 @@ export class MailArchiveGitHelper {
...
             const whatsCookingBaseURL = "https://lore.kernel.org/git/";
-            for (const branchName of Object.keys(sousChef.branches)) {
+            for (const branchName of sousChef.branches.keys()) {
```
Comment (webstech) [inline lib/mail-archive-helper.ts:122]: Sorry I missed this (I had updated in my local test branch).  May need to look at creating tests for `MailArchiveGitHelper`.

Comment (dscho) [reply lib/mail-archive-helper.ts:122]: That would be nice. I am not quite comfortable yet adding these tests...

**Comment by dscho** [comment]
So... https://lore.kernel.org/git/xmqqft1542wu.fsf@gitster.c.googlers.com/ has been sent on Monday, and it looks as if this PR did its job (see e.g. https://github.com/gitgitgadget/git/pull/889#issuecomment-793188741)!

**Pull request merged**